images/scripts/fedora-34.setup

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
```

Comment:
This file should be a symlink to fedora.setup, not a copy, like fedora-{32,33}.setup. Let's please avoid divergence, and put f-34 special cases right into the setup.